apps/account-api/src/services/siwfV2.service.spec.ts

```diff
@@ -239,50 +295,56 @@ describe('SiwfV2Service', () => {
     });
 
     describe('getSiwfV2LoginResponse', () => {
-      it('Should parse the control key', async () => {
-        const result = await siwfV2Service.getSiwfV2LoginResponse(validSiwfAddDelegationResponsePayload);
-
-        expect(result).toBeDefined();
-        expect(result.controlKey).toEqual('f6akufkq9Lex6rT8RCEDRuoZQRgo5pWiRzeo81nmKNGWGNJdJ');
-      });
-
-      it('Should parse the email', async () => {
-        const result = await siwfV2Service.getSiwfV2LoginResponse(validSiwfAddDelegationResponsePayload);
-
-        expect(result).toBeDefined();
-        expect(result.email).toEqual('john.doe@example.com');
-      });
-
-      it('Should parse the phone number', async () => {
-        const result = await siwfV2Service.getSiwfV2LoginResponse(validSiwfAddDelegationResponsePayload);
-
-        expect(result).toBeDefined();
-        expect(result.phoneNumber).toEqual('+01-234-867-5309');
-      });
-
-      it('Should parse the graph key', async () => {
-        const result = await siwfV2Service.getSiwfV2LoginResponse(validSiwfAddDelegationResponsePayload);
-
-        expect(result).toBeDefined();
-        expect(result.graphKey).toBeDefined();
-        expect(result.graphKey.encodedPrivateKeyValue).toEqual(
-          '0xd0910c853563723253c4ed105c08614fc8aaaf1b0871375520d72251496e8d87',
-        );
-        expect(result.graphKey.encodedPublicKeyValue).toEqual(
-          '0xb5032900293f1c9e5822fd9c120b253cb4a4dfe94c214e688e01f32db9eedf17',
-        );
-      });
-
-      it('Should parse MSA Id', async () => {
-        jest.spyOn(blockchainService, 'publicKeyToMsaId').mockResolvedValueOnce('123456');
-
-        const result = await siwfV2Service.getSiwfV2LoginResponse(validSiwfAddDelegationResponsePayload);
-
-        expect(result).toBeDefined();
-        expect(result.msaId).toEqual('123456');
-        expect(blockchainService.publicKeyToMsaId).toHaveBeenCalledWith(
-          'f6akufkq9Lex6rT8RCEDRuoZQRgo5pWiRzeo81nmKNGWGNJdJ',
-        );
+      const email = 'john.doe@example.com';
```

Comment:
refactored this so I could try all the test cases with both ss58 and ethereum keys.